pkg/porter/explain.go

```diff
@@ -0,0 +1,291 @@
+package porter
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/deislabs/cnab-go/bundle"
+	"github.com/deislabs/porter/pkg/context"
+	"github.com/deislabs/porter/pkg/printer"
+	"github.com/olekukonko/tablewriter"
+	"github.com/pkg/errors"
+)
+
+type ExplainOpts struct {
+	BundleLifecycleOpts
+	printer.PrintOptions
+}
+
+// PrintableBundle holds a subset of pertinent values to be explained from a bundle.Bundle
+type PrintableBundle struct {
+	Name        string                         `json:"name" yaml:"name"`
```

Comment:
I was looking more closely at https://github.com/ghodss/yaml as a replacement for the standard yaml encoder and one nice perk is that you don't need to do double tag duty. It drives itself off of the json tags and marshalers which is pretty neat. Maybe something we could think about doing later.

Reply:
Sorry for the drive-by on your PR. I just thought you'd be interested in this after seeing the double tags. I've turned this into an issue for us to think about. https://github.com/deislabs/porter/issues/650

Reply:
That sounds good!